.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Proof of Leadership/circomlib"]
```

Comment:
Can we rename this folder to `proof_of_leadership`?

Don't use spaces in fs paths

Reply:
Done

Reply:
is it correct now?

Reply:
looks right!